sample_data_config/semantic-scholar/semantic-scholar-recommendation.config.yaml

```diff
@@ -0,0 +1,86 @@
+semanticScholarRecommendation:
+  - dataPipelineId: semantic_scholar_recommendation_data_pipeline_1
+    matrix:  # similar to ci matrix, currently supporting a single variable only
+      list:
+        include:
+          bigQuery:
+            projectName: 'elife-data-pipeline'
+            # the SQL should provide:
+            # - list_key (any identifier)
+            # - list_meta (any metadata, will get saved with the response)
+            # - list, array with the following fields:
+            #   - doi
+            #   - is_excluded
+            # using "staging" dataset for end-to-end test as it will contain some data 
+            sqlQuery: |-
+              SELECT *
```

Comment:
This query is a bit long. It was an attempt to decouple this pipeline from the Sciety events database structure.